examples/input_files/reference_SAOCOM.xml

```diff
@@ -0,0 +1,46 @@
+<component name="reference">
+    <property name="IMAGEFILE">
+        S1A_OPER_SAR_EOSSP__CORE_L1A_OLF_20200610T203550/Data/slc-acqId0000076824-a-sm8-0000000000-s8dp-vv
+    </property>
+    <property name="XEMTFILE">
+        S1A_OPER_SAR_EOSSP__CORE_L1A_OLF_20200610T203550.xemt
+    </property>
+    <property name="XMLFILE">
+        S1A_OPER_SAR_EOSSP__CORE_L1A_OLF_20200610T203550/Data/slc-acqId0000076824-a-sm8-0000000000-s8dp-vv.xml
+    </property>
+    <property name="OUTPUT">
+        slc-acqid0000076824-a-sm8-s8dp-vv.slc
+    </property>
+    <property name="unwrapper name">snaphu</property> 
```

Comment:
This unwrapper name entry here is probably a typo.

Reply:
Hi, I left that tag, since it works better for me than the unwrapper that runs by default.